packages/nextcloud/lib/src/webdav/client.dart

```diff
@@ -146,23 +146,23 @@ class WebDavClient {
   ///   * [putStream] for a complete operation executing this request.
   http.BaseRequest putStream_Request(
     Stream<List<int>> localData,
+    int contentLength,
```

Comment:
I'd like to leave this as a named parameter.
1. easier to read code
2. less breaking for our users